services/director-v2/setup.cfg

```diff
@@ -6,3 +6,6 @@ tag = False
 commit_args = --no-verify
 
 [bumpversion:file:VERSION]
+
+[tool:pytest]
+asyncio_mode = auto
```

Comment:
why was this necessary? it is already in the makefile right?

Reply:
is it? I need to double check then. There was a warning in the tests ...

Reply:
But yes, I see that the scripts in ``ci`` and the ``makefile`` recipes incorporate this option in CLI
That is strange then. Perhaps I was running ``pytest`` manually via CLI.